examples/panel/chat-with-pdf/requirements.txt

```diff
@@ -1,6 +1,137 @@
-panel
-openai
-chromadb
-pypdf
-langchain
-tiktoken
\ No newline at end of file
+aiohappyeyeballs==2.4.3
+aiohttp==3.9.3
+aiosignal==1.3.1
+annotated-types==0.6.0
+anyio==4.3.0
+asgiref==3.7.2
```

Comment:
let's pin versions but only for the explicitly imported packages

Reply:
Fix: [3a3a901](https://github.com/ploomber/doc/pull/284/commits/3a3a9018cab02eeef5c5fe28847481d41929a066)